features/pom.xml

```diff
@@ -62,12 +62,12 @@
         <geronimo-jta-spec-version>1.1.1</geronimo-jta-spec-version>
         <camel.guava.version>19.0</camel.guava.version>
         <cxf.jackson.version>${jackson.version}</cxf.jackson.version>
-        <cxf.jackson.databind.version>${jackson.databind.version}</cxf.jackson.databind.version>
+        <cxf.jackson.databind>${jackson.databind}</cxf.jackson.databind>
```

Comment:
You can remove this too. All dependencies should use ${jackson.version} or ${cxf.jackson.version}

In fact I don't think you have ${jackson.databind} defined anywhere.

Reply:
Yeah that's the reason for the build failure ^

[2020-05-22T14:52:33.469Z] [ERROR] [ERROR] Some problems were encountered while processing the POMs:
[2020-05-22T14:52:33.469Z] [ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.core:jackson-databind:jar must be a valid version but is '${jackson.databind}'. @ line 102, column 22
[2020-05-22T14:52:33.469Z] [ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.core:jackson-databind:jar must be a valid version but is '${jackson.databind}'. @ line 373, column 22
[2020-05-22T14:52:33.469Z] [ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.core:jackson-databind:jar must be a valid version but is '${jackson.databind}'. @ line 52, column 22